app/views/procedures/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<% @procedures.each do |p|%>
```

Comment:
You aren't passing `@procedures` to this view in the Procedures controller.

Reply:
Deleted this page, will work on it from scratch later